src/iop/filmic.c

```diff
@@ -253,8 +301,9 @@ void init_presets(dt_iop_module_so_t *self)
   // Presets indoors
   p.grey_point_source = 18.0f;
   p.preserve_color = 0;
-  p.latitude_stops = 2.0f;
+  p.latitude_stops = 2.0f;git
```

Comment:
I suppose "git" should have been put in the console :)

Reply:
Obviously ^^